src/uct/sm/mm/base/mm_iface.c

```diff
@@ -138,10 +138,17 @@ uct_mm_iface_is_reachable_v2(const uct_iface_h tl_iface,
     }
 
     iface_addr = (void*)params->iface_addr;
+    if (iface_addr == NULL) {
+        uct_iface_fill_info_str_buf(params, "iface address is empty");
+        return 0;
+    }
+
+    if (!uct_mm_md_mapper_ops(md)->is_reachable(md, iface_addr->fifo_seg_id,
```

Comment:
What's the reason for this change? Seems the logic is the same (`return A && B && C`)

Reply:
You're right, I initially changed it so I could write an error message to the string but eventually I did it internally and didn't switch it back the original version